### ISSUE TITLE
Send message overtime

### ISSUE THREAD

**Issue by shukamu95** [vin350/TwitchPointsFarmer#5]
Can your add feature to accounts send predefined message (from txt file) from random user with timer?

**Comment by Agentew04** [comment]
Like a single txt file or schedule many messages in various times of the day? Or like a message every x hours/minutes?

**Comment by shukamu95** [comment]
Hey @Agentew04 thank you for quick response.
 
A single txt file with messages per line is sufficient. Like:
!points
!shop
!commands

And send message every X minutes, if the program can select a random timer looks better for real chat simulation.
Example: Timer between 5~10min and select a random minutage to send the message.

If the program can have a trigger to send determined message you're godlike haha.
Example:
Real user say something in chat like: wow good play
Bot's send emotes or determined message.

I can pay for it as you will invest your time to code this.

ps: você é br? haha

**Comment by Agentew04** [comment]
> ps: você é br? haha

Sim, kkkkk. Mas acho q tem como fazer sim, talvez um arquivo com formatação especial para os timer, etc. Tem uma proposta em mente sobre o pagamento?
